cluster/aws.go

```diff
@@ -1090,7 +1081,7 @@ func (c *AWSCluster) GetClusterDetails() (*components.ClusterDetailsResponse, er
 	log := logger.WithFields(logrus.Fields{"tag": "GetClusterDetails"})
 	log.Info("Start getting cluster details")
 
-	c.GetK8sConfig()
+	c.DownloadK8sConfig()
```

Comment:
This is wrong here because https://github.com/banzaicloud/pipeline/pull/538/files#diff-8e624ff87723b97e43c3758eff0c292dL828 is factored out so this method does nothing here.